src/app/Shared/Services/Login.service.tsx

```diff
@@ -85,7 +85,7 @@ export class LoginService {
       });
   }
 
-  checkAuth(token: string, method: string, rememberMe = false): Observable<boolean> {
+  checkAuth(token: string, method: string, rememberMe = true): Observable<boolean> {
```

Comment:
Even with this change, refreshing the frontend generates an additional access token every time. The network tab shows an additional auth query somewhere that sends an empty token, which prompts the backend to send the login redirect url. I'll keep looking into this.

Reply:
Should be fixed now